### ISSUE TITLE
Improve Grid footer cell accessibility and keyboard navigation.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#923]
Improve Grid footer cell accessibility and keyboard navigation.

The footer cells in the Grid should be accessible for screen readers. The user should be able to move from the data rows to the footer rows.

The Grid built-in navigation should be able to go over the footer rows.
